countrycodepicker/src/main/java/com/rilixtech/widget/countrycodepicker/CountryCodeArrayAdapter.java

```diff
@@ -66,14 +67,13 @@ private void setData(Country country, ViewHolder viewHolder) {
     viewHolder.tvCode.setVisibility(View.VISIBLE);
     viewHolder.llyFlagHolder.setVisibility(View.VISIBLE);
     Context ctx = viewHolder.tvName.getContext();
-    String name = country.getName();
     String iso = country.getIso().toUpperCase();
     String countryNameAndCode;
 
     if (mCountryCodePicker.isHideNameCode()) {
-      countryNameAndCode = name;
+      countryNameAndCode = new Locale(Locale.getDefault().getLanguage(), iso).getDisplayCountry();
```

Comment:
It'd be better to create the default locale instance in the class instead of creating a new instance in every call - especially as this method gets called for every country.

Reply:
I meant to extract `Locale.getDefault().getLanguage()` into a class level variable

Reply:
Yeah got it!